Addons/tsgExoticQuadrature.hpp

```diff
@@ -104,17 +116,24 @@ inline double lagrange_eval(size_t idx, const std::vector<double> &roots, double
     return eval_value;
 }
 
-//! \brief Generate the full cache of Exotic (weighted and possibly shifted) quadrature weights and points for an (input) max level
-//! \b n, using parameters \b shift, \b weight_fn_vals, \b shifted_weights, \b ref_points, and \b nref.
-//!
-//! The integral  I := ∫ F(x) * weight_fn(x) dx, for a given level, is approximated as:
-//! I ≈ sum{f(roots[i]) * shifted_weights[i]} - shift * sum{f(correction_points[i]) * correction_weights[i]}.
-//!
-//! It is assumed that shifted_weights[i] is nonnegative and it is computed as shifted_weights[i] = ref_weights[i] *
-//! (weight_fn_vals[i] + shift) for every i, where ref_weights are the quadrature weights corresponding to ref_points.
-inline TasGrid::CustomTabulated getExoticQuadrature(const int n, const double shift, const std::vector<double> &weight_fn_vals,
-                                                    const std::vector<double> &shifted_weights, const std::vector<double> &ref_points,
-                                                    const int nref, const char* description, const bool is_symmetric = false) {
+
+/*!
+ * \internal
+ * \ingroup TasmanianAddons
```

Comment:
Add a new Doxygen group `TasmanianAddonsExoticQuad` and use that group for all methods.

Reply:
The mathematical explanation of exotic quadrature means should not be internal, it should be part of the description for the group.

Reply:
> Add a new Doxygen group `TasmanianAddonsExoticQuad` and use that group for all methods.

Done.

> The mathematical explanation of exotic quadrature means should not be internal, it should be part of the description for the group.

Moved this to the newly created Doxygen group.